echo_handlers.go

```diff
@@ -459,5 +409,18 @@ func (svc *Service) LogoutHandler(c echo.Context) error {
 		sess.Options.Domain = svc.cfg.CookieDomain
 	}
 	sess.Save(c.Request(), c.Response())
-	return c.Redirect(302, "/")
+	return c.JSON(http.StatusOK, "Logout successful")
+}
+
+func (svc *Service) InfoHandler(c echo.Context) error {
```

Comment:
I am not sure about this endpoint.
- CSRF handling - why is it done here?
- returning the user in an "info"? this is not really consistent with our other APIs